Source/Core/DiscIO/DiscUtils.h

```diff
@@ -49,6 +50,24 @@ constexpr u32 WII_NONPARTITION_DISCHEADER_SIZE = 0x100;
 constexpr u32 WII_REGION_DATA_ADDRESS = 0x4E000;
 constexpr u32 WII_REGION_DATA_SIZE = 0x20;
 
+// Default block size for GCZ/RVZ images
+constexpr int GCZRVZ_DEFAULT_BLOCK_SIZE = 0x20000; // 128 KiB
+
+// Because DVD timings are emulated as if we can't read less than an entire ECC block at once
+// (32 KiB - 0x8000), there is little reason to use a block size smaller than that.
+constexpr int GCZRVZ_MIN_BLOCK_SIZE = 0x8000; // 32 KiB
+
+// For performance reasons, blocks shouldn't be too large.
+// 2 MiB (0x200000) was picked because it is the smallest block size supported by WIA.
+constexpr int GCZRVZ_MAX_BLOCK_SIZE = 0x200000; // 2 MiB
```

Comment:
This applies to WIA in the exact same way as it applies to GCZ and RVZ, so the name should be changed.

Perhaps consider naming it something like `RECOMMENDED_MAX_BLOCK_SIZE`, since larger block sizes do in fact work fine, to better contrast it with `WIA_MIN_BLOCK_SIZE` (which in fact is a strict limit).